internal/host/common.go

```diff
@@ -19,17 +19,18 @@ import (
 )
 
 const (
-	statusInfoDisconnected               = "Host keepalive timeout"
-	statusInfoDisabled                   = "Host is disabled"
-	statusInfoDiscovering                = "Waiting for host hardware info"
-	statusInfoInsufficientHardware       = "Host does not pass minimum hardware requirements"
-	statusInfoPendingForInput            = "User input required"
-	statusInfoNotReadyForInstall         = "Host not ready for install"
-	statusInfoInstalling                 = "Installation in progress"
-	statusInfoResettingPendingUserAction = "Reboot the host into the installation image to complete resetting the installation"
-	statusInfoPreparingForInstallation   = "Preparing host for installation"
-	statusInfoPreparingTimedOut          = "Cluster is no longer preparing for installation"
-	statusInfoAbortingDueClusterErrors   = "Installation has been aborted due cluster errors"
+	statusInfoDisconnected               = "Host has stopped communicating with the installation service"
+	statusInfoDisabled                   = "Host was disabled by user"
+	statusInfoDiscovering                = "Waiting for host to send hardware details"
+	statusInfoInsufficientHardware       = "Host does not possess minimum hardware requirements"
+	statusInfoPendingForInput            = "Waiting for user input"
+	statusInfoNotReadyForInstall         = "Host cannot be installed due to failing validation(s)"
+	statusInfoKnown                      = "Host is ready to be installed"
```

Comment:
Out of scope for this PR, but would it make sense for this one to be `statusInfoReadyForInstall` and the UI would show `Ready`? Seeing all hosts in the table as "Ready" and then clicking Next/Install may be nice, rather than the user maybe having to click the `Known` status to understand that it means "Ready to install".

Reply:
Changing the name of the status from "known" to "ready" would be a pretty big change for us.  Maybe just change it in the UI?

Reply:
Makes sense, I figured it was out of scope 🙂. WDYT about a UI-only change @nirfarkas @jtomasek? I know in general we like to reflect the API so that the CLI is consistent too.